packages/cli/src/commands/studioHandler.js

```diff
@@ -18,7 +18,7 @@ export const handler = async (options) => {
 
     // Import studio and start it
     const { serve } = await import('@redwoodjs/studio')
-    await serve({ open: options.open })
+    await serve({ open: options.open, enableWeb: true })
```

Comment:
Thanks for the heads-up 🙂 But as you say – it works with Studio as it is now. When we upgrade Studio to use the latest server file stuff this will have to change too.